andela/rivendell-ah-client
# #160056373 Users should be able to recieve and use a reset password link

#### What does this PR do?
- This PR will enable a user to receive a reset password link and update his/her password.

#### Description of Task to be completed?
- provide forget password links to be added to`/ login` and `/signup` page
- create a page `/forgot-password` where a user can enter the email to receive the reset password link
- ensure user receives an email
- redirect a user to `/reset-password` page with the form to update a password.
- update a user password when he provides a valid password
- show the error messages when a password is not valid or not updated

#### How should this be manually tested?
- clone repo and install dependences with `npm install`
- `npm run dev` to run locally
- navigate to `/forgot-password` to enter an email and send link
- check email and click on the reset password link
- enter the new password and on the provided form

#### What are the relevant pivotal tracker stories?
#160056373

![image](https://user-images.githubusercontent.com/39072402/45635826-f18d4800-ba9d-11e8-9d21-7b4d362c6832.png)
<img width="418" alt="screen shot 2018-09-17 at 5 17 55 pm" src="https://user-images.githubusercontent.com/39072402/45636405-893f6600-ba9f-11e8-8651-940bc0553edd.png">
<img width="419" alt="screen shot 2018-09-17 at 5 17 04 pm" src="https://user-images.githubusercontent.com/39072402/45636406-893f6600-ba9f-11e8-9d43-1780477597d6.png">
<img width="416" alt="screen shot 2018-09-17 at 5 15 01 pm" src="https://user-images.githubusercontent.com/39072402/45636407-893f6600-ba9f-11e8-90c3-6eafc1bf8c90.png">
<img width="414" alt="screen shot 2018-09-17 at 5 14 28 pm" src="https://user-images.githubusercontent.com/39072402/45636408-89d7fc80-ba9f-11e8-819a-c70a37c99d9d.png">
<img width="424" alt="screen shot 2018-09-17 at 5 13 38 pm" src="https://user-images.githubusercontent.com/39072402/45636410-89d7fc80-ba9f-11e8-9e14-afa3a27dc082.png">
<img width="583" alt="screen shot 2018-09-17 at 5 12 46 pm" src="https://user-images.githubusercontent.com/39072402/45636411-89d7fc80-ba9f-11e8-9443-1e318947d240.png">
